lib/workers/repository/dependency-dashboard.ts

```diff
@@ -350,7 +350,7 @@ export async function ensureDependencyDashboard(
       { title: config.dependencyDashboardTitle },
       'DRY-RUN: Would ensure Dependency Dashboard'
     );
-  } else {
+  } else if (!GlobalConfig.get('dryRun')) {
```

Comment:
I'm not sure this is necessary. See screenshot: 
![image](https://user-images.githubusercontent.com/6311784/160625978-366fbf9d-11dd-4159-ae6e-e0133c53d947.png)